django_dramatiq/models.py

```diff
@@ -10,9 +12,19 @@
 #: The database label to use when storing task metadata.
 DATABASE_LABEL = DjangoDramatiqConfig.tasks_database()
 
+DJANGO_DRAMATIQ_TASKS_NOT_WRITES = getattr(settings, "DJANGO_DRAMATIQ_TASKS_NOT_WRITES", [])
+DJANGO_DRAMATIQ_TASKS_WRITES_ONLY = getattr(settings, "DJANGO_DRAMATIQ_TASKS_WRITES_ONLY", [])
+
 
 class TaskManager(models.Manager):
-    def create_or_update_from_message(self, message, **extra_fields):
+    def create_or_update_from_message(self, message, **extra_fields) -> Optional['Task']:
+
+        # black and write lists
+        if DJANGO_DRAMATIQ_TASKS_WRITES_ONLY and message.actor_name not in DJANGO_DRAMATIQ_TASKS_WRITES_ONLY:
```

Comment:
**suggestion:** Since you default to an empty list in the getattr method above, the first part of this conditional should not be necessary. 
```suggestion
        if message.actor_name not in DJANGO_DRAMATIQ_TASKS_WRITES_ONLY:
```

Reply:
DJANGO_DRAMATIQ_TASKS_WRITES_ONLY = False-like vals will not works this way. For example - None